### ISSUE TITLE
[Bug]: Event page does not match design

### ISSUE THREAD

**Issue by katjam** [GenderedIntelligence/the-trans-dimension#278]
## Design notes
The following bits are mostly about spacing, which affects how folks absorb information and will impact how easy it is to absorb key info like contact information etc.

Event page [All states]
- The weight of the sponsor name text / link is wrong
- Spacing between the end of the contact information and the end of the box is too small.
- When an event address is not provided it should say something
- Online hangouts are showing as physical addresses not links etc

## Acceptance Criteria
- [ ] The font weight of sponsor name link matches XD designs

**Comment by katjam** [comment]
@studiosquid 

Shall we...
1) add to the bottom of work queue, 2) close as a won't fix, 3) leave open for future niceness but not important now.

**Comment by studiosquid** [comment]
@katjam I think 1, these look like very easy fixes the font weight is a number change? (Fingers crossed it doesn't change the weight of the entire site... if it does and that becomes a whole thing, then drop it.)  

I also wonder if the spacing issue is the same spacing issue which was flagged in another issue — so they might fix both? https://github.com/GenderedIntelligence/the-trans-dimension/issues/358

**Comment by katjam** [comment]
These are the XD files from 2022 which might help. May be this work has been done but see if you can find the discrepancy mentioned and go from there.

for desktop: https://xd.adobe.com/view/f1786926-b132-4a19-8e22-b9cf5dbe7362-a42b/
for mobile: https://xd.adobe.com/view/21a46e92-cce4-4007-b7c6-a3ca9b7d93ee-74f7/

@james-collapse are you working on this? I thought it might be a nice small one for @ponsillius if not.

**Comment by james-collapse** [comment]
I’m not, @ponsillius feel free to pick this up!

**Comment by katjam** [comment]
@studiosquid Do you know which bit of text is the wrong weight? Referred to here as "sponsor name". It's hard to tell from the XD designs attached above whether it's maybe that the partner name should be bold?

![image](https://github.com/user-attachments/assets/ca0da7ba-620f-4f85-a628-52e0c0816d6a)


It may be that this issue has gone away.

**Comment by studiosquid** [comment]
@katjam yes it's the weight, it should be "Semibold italic" rather than Medium italic — and still looks quite different in my XD file and the preview — I had a quick tinker in browser — font weight should be 600 rather than 500 there. (It's not a huge difference but it helps in absorbing the information.)
